extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -201,9 +204,13 @@ export class LinkerManager {
 
     const optInMeta = this.ampdoc_.win.document.head./*OK*/querySelector(
         'meta[name="amp-google-client-id-api"][content="googleanalytics"]');
-    const isGaType = this.type_ === 'googleanalytics';
+    if (optInMeta.hasAttribute(LINKER_CREATED) ||
```

Comment:
Potential NPE here. please fix and add a test.

Reply:
Yikes. I wasn't sure why the existing tests did not catch this, but it is because it early exits if experiment is not on. I turned on the experiment for all the tests. Should we go ahead and remove the experiment in a follow up PR?

Reply:
yep, let's remove the experiment in next PR